sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutValidationTests.java

```diff
@@ -196,7 +310,7 @@ public void clientLevelEndToEndTimeoutPolicyInOptionsShouldTimeout() {
                 "name123",
                 2,
                 UUID.randomUUID().toString());
-            CosmosItemResponse response = container.createItem(obj).block();
+            CosmosItemResponse<?> response = container.createItem(obj).block();
```

Comment:
nit: The change can be removed altogether since `response` is unused.

Reply:
Fixed